server/cmd/mmctl/commands/import.go

```diff
@@ -346,6 +346,10 @@ func jobListCmdF(c client.Client, command *cobra.Command, jobType string) error
 	if err != nil {
 		return err
 	}
+	if perPage > MaxPageSize {
+		perPage = MaxPageSize
+		printer.PrintError(fmt.Sprintf("Per page value is greater than the maximum allowed. Reduced to %d", MaxPageSize))
+	}
```

Comment:
I wonder if we could generalize your solution and apply it to the `per-page` arguments in other commands while keeping the code DRY. Does that sound double to you?

Reply:
@hanzei Do all the other commands have the same server-side per-page maximum?

Reply:
@hanzei So it looks like the per-page maximum is applied via the `ParamsFromRequest` function, so it would appear to effect all the commands in the same way. That's good for generalizing what I did.

My next question is where a generalized, DRY function would live on the client side. As it is, the only shared code across all commands is `Client` (which seems dedicated to defining the functions for accessing API endpoints) and `Printer` (which handles the outputting of messages to the user's terminal). Neither seem appropriate for placing a generalized utility function. Should I make a new utility package to hold this one function?